### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gfwilliams**
Proposing some doc changes based on https://github.com/sandeepmistry/bleno/issues/24#issuecomment-252669205

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ __Note:__ Mac OS X, Linux, and Windows are currently the only supported OSes.
       * ```sudo systemctl stop bluetooth``` (once)
       * ```sudo systemctl disable bluetooth``` (persist on reboot)
 
+If you're using BlueZ 5.14 or later, you may find that running noble *and* bleno at the same time stops you from retrieving the list of services over BLE.
+
```
Comment (sandeepmistry) [inline README.md]: For using both at the same time there was a new env. var introduced in noble (https://github.com/sandeepmistry/noble/pull/379) to allow both to work together in the same process.

Comment (sandeepmistry) [inline README.md]: That was for a single process.

Can you please try again with the latest release of noble and bleno? There was a fix in https://github.com/sandeepmistry/node-bluetooth-hci-socket/commit/14dc3660f9e86695c6c75b92dc2222da2905549e which was released with `bluetooth-hci-socket` v0.5.1, both noble and bleno are using this release in the latest package.json's.

Comment (gfwilliams) [inline README.md]: Thanks - sorry about this. It's one adaptor - I'll try again with NOBLE_MULTI_ROLE and see if that helps.

Comment (gfwilliams) [inline README.md]: Great - seems to work, thanks! Is it worth me adding that, or shall I just strip it back to the single link back to prerequisites?

Comment (sandeepmistry) [inline README.md]: Great news, thanks for trying it out!

Yes, I definitely think we should add this info, maybe also [a link back to the noble read me](https://github.com/sandeepmistry/noble#bleno-compatibility). Do we still need to new part about BlueZ or does it just apply to bleno?

Comment (gfwilliams) [inline README.md]: Ahh, that looks great. Can't believe I missed that (I must admit I only looked in Bleno's docs, not Noble - but I guess others may do the same). I've just pushed a documentation change to link to it - but feel free to move it around or cancel the PR and make your own changes :)

**Comment by sandeepmistry** [comment]
@gfwilliams thank you, suggested changes look great!

**Pull request merged**